src/dask_dirac/_dask.py

```diff
@@ -41,20 +42,27 @@ def __init__(
         if user_proxy is None:
             user_proxy = "/tmp/x509up_u1000"
         if jdl_file is None:
-            jdl_file = getcwd() + "/grid_JDL"
+            jdl_file = "/tmp/dask-dirac-JDL"
```

Comment:
When creating temporary files, it is good practise to randomise part of the name, e.g.

```python
import hashlib
import getpass

def string_to_short_hash(string: str) -> str:
    """Convert a string to a short hash."""
    return hashlib.sha1(string.encode()).hexdigest()[:8]
    
jdl_file = "/tmp/dask-dirac-JDL" + "_" + string_to_short_hash(getpass.getuser())
```

which creates something like `/tmp/dask-dirac-JDL_da606130` - unique to each user.

Of course, you could also just use the username, but that function will be useful for other things: e.g. caches.